src/views/BottomTabBar.js

```diff
@@ -268,6 +283,15 @@ const styles = StyleSheet.create({
     borderTopWidth: StyleSheet.hairlineWidth,
     borderTopColor: 'rgba(0, 0, 0, .3)',
     flexDirection: 'row',
+    ...Platform.select({
+      web: {
+        position: 'fixed',
```

Comment:
This should probably be `position: 'absolute'`. When the tab view doesn't fill the whole screen, we don't wanna position things relative to the viewport (e.g. a sidebar on left and the tab view on right).